spec/services/doc_auth/lexis_nexis/requests/true_id_request_spec.rb

```diff
@@ -81,42 +79,104 @@
         end
       end
     end
+
+    def include_liveness
+      return false if Identity::Hostdata.env == 'prod'
+      return false unless IdentityConfig.store.doc_auth_selfie_capture_enabled
+
+      liveness_checking_required
+    end
```

Comment:
Do we need to repeat this method definition in the spec?

Reply:
@soniaconnolly - it allows us to reduce _a lot_ of lines of coding in this test .... do you have any simple suggestions on how to improve this?

Reply:
Paired on this and we ended up renaming the method to include_liveness_expected to make it clear that it's not overriding the method definition in code.